lib/dynamic_widget/basic/readmoretext_widget_parser.dart

```diff
@@ -2,9 +2,15 @@ import 'package:dynamic_widget/apnaklub_widgets/readmoretext.dart';
 import 'package:dynamic_widget/dynamic_widget.dart';
 import 'package:flutter/widgets.dart';
 
+import '../../new_widget_parser.dart';
 import '../utils.dart';
 
 class ReadMoreTextParser extends NewWidgetParser {
+  @override
+  void assertionChecks(Map<String, dynamic> map) {
+    // TODO: implement assertionChecks
```

Comment:
please add assertions here

Reply:
@Suryapratap-R  please check new changes and also go through added TODO in readmoretext_widget_parser.dart